contracts/OutflowLimitedAMMSV.sol

```diff
@@ -69,18 +69,34 @@ contract OutflowLimitedAMMSV is AccessManagedMSV {
     emit LimitChanged(slotSize, limit);
   }
 
+  /**
+   * @dev Returns the current time slot size in seconds.
+   */
   function getOutflowLimitSlotSize() external view returns (uint256) {
     return _getLOMStorage().slotSize;
   }
 
+  /**
+   * @dev Returns the current withdraw limit for each slot.
```

Comment:
`Returns the net outflow limit that will be applied on two consecutive time slots`